distribution/changelog.txt

```diff
@@ -15,6 +15,7 @@
 - Fix: [#12820] Title menu buttons not invalidating properly
 - Fix: [#12845] Deleting ride with active ad campaign creates incorrect notification.
 - Fix: [#12857] Incorrect Peep thoughts in imported RCT1 parks.
+- Fix: [#12881] Guests have no favorite ride.
```

Comment:
This suggests the favourite ride is missing entirely, implying that save games are affected. Since this is a UI issue, I suggest we phrase it as such, e.g.:
```suggestion
- Fix: [#12881] Guests' favourite rides are not listed in the guest window.
```